src/interactions/Commands/ChatInputCommands/devFunctions.ts

```diff
@@ -0,0 +1,476 @@
+				},
+				{
+					name: "command",
+					description: "La commande à exécuter",
+					type: ApplicationCommandOptionType.String,
+					required: true,
+					autocomplete: true,
+				},
+			],
+			execute: async (interaction: ChatInputCommandInteraction) => {
+				let commandPath = interaction.options.getString("command", true);
+				let { command, subCommandGroup, subCommand } = (getFullCommandPath.exec(commandPath)?.groups as CommandNames) ?? {
+					groups: { command: "", subCommandGroup: "", subCommand: "" },
+				};
+				if (!localCommands.has(command)) return interaction.editReply("Cette commande n'existe pas");
+				let localCommand = localCommands.get(command) as ChatInputApplicationCommandData;
+				let subCommandGroupData: ApplicationCommandSubGroupData | undefined;
+				let subCommandData: ApplicationCommandSubCommandData | undefined;
+				let executableCommand:
+					| ChatInputApplicationCommandData
+					| ApplicationCommandSubCommandData
+					| ApplicationCommandSubGroupData
+					| undefined;
+				if (subCommandGroup) {
+					subCommandGroupData = localCommand.options?.find(
+						(option: { name: string | undefined }) => option.name === subCommandGroup
+					) as ApplicationCommandSubGroupData;
+					if (!subCommandGroupData) return interaction.editReply("Cette commande n'existe pas");
+				}
+				subCommandData = (subCommandGroupData || localCommand).options?.find(
+					(option: { name: string | undefined }) => option.name === subCommand
+				) as ApplicationCommandSubCommandData;
+				if (!!subCommandData && subCommandData.execute) {
+					executableCommand = subCommandData;
+				} else {
+					executableCommand = localCommand;
+				}
+
+				const optionsMap = await collectVariables(interaction, executableCommand.options);
+				const options: OptionAndValue[] = [];
+				for (const [, value] of optionsMap) {
+					options.push(value);
+				}
+
+				const interactionJson = interaction.toJSON() as { [key: string]: any };
+				interactionJson.options = interaction.options.data;
+				const newInteraction = (await getNewInteraction(
+					interaction,
+					{ command, subCommandGroup, subCommand },
+					options
+				)) as ChatInputCommandInteraction;
+				try {
+					localCommand.execute(newInteraction);
+				} catch (e) {
+					console.error(e);
+					interaction.editReply("Une erreur est survenue");
+				}
+			},
+		},
+	],
+
+	async execute(interaction: ChatInputCommandInteraction): Promise<void> {
+		await interaction.deferReply({ ephemeral: true });
+		let subCommand = interaction.options.getSubcommand(true);
+		let choosenSubcommand = this.options!.find((option: { name: any }) => option.name === subCommand) as ApplicationCommandSubCommandData;
+		!!choosenSubcommand && !!choosenSubcommand.execute && choosenSubcommand.execute(interaction);
+	},
+
+	async onAutocomplete(interaction: AutocompleteInteraction) {
+		let focusedInteraction = interaction.options.getFocused(true);
+		let value = focusedInteraction.value;
```

Comment:
rename "value" to make it more understandable that we are talking of current user input